app/controllers/packages_controller.rb

```diff
@@ -0,0 +1,73 @@
+require 'httparty'
+
+class PackagesController < ApplicationController
+ # include Timeout
+
+  def rates
+    origin = get_origin
+    package = get_package
+    destination = get_destination
```

Comment:
Love that you pulled out this logic into their own methods. Single responsibility! :tada: